src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -65,14 +69,33 @@ fun List<OppfolgingstilfelleDag>.gradertAtTilfelleEnd() =
         it.priorityOppfolgingstilfelleBit != null
     }.isGradert()
 
-fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle() =
-    Oppfolgingstilfelle(
+fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle(): Oppfolgingstilfelle {
+    val onlySykmeldingNyOrInntektsmelding = this.all { dag ->
+        dag.priorityOppfolgingstilfelleBit?.tagList
+            ?.let { tagList ->
+                tagList in ((Tag.SYKMELDING and Tag.NY) or (Tag.INNTEKTSMELDING and Tag.ARBEIDSGIVERPERIODE))
+            }
+            ?: false
+    }
+    if (onlySykmeldingNyOrInntektsmelding && this.durationDays() > 118) {
+        val sampleUUID = this.mapNotNull { dag -> dag.priorityOppfolgingstilfelleBit }.firstOrNull()?.uuid
```

Comment:
IntelliJ foreslår å forenkle til `this.firstNotNullOfOrNull { dag -> dag.priorityOppfolgingstilfelleBit }?.uuid` 🤯 
Og vil vi logge og telle hvis `sampleUUID` er `null`?

Reply:
Føler meg ganske sikker på at det aldri vil forekomme at sampleUUID er null, men skulle det være tilfelle er det like greit å logge.